utils/mvcli.go

```diff
@@ -338,6 +338,8 @@ func (m *Mvcli) processDriveType(pdType, ssdType string) string {
 		return common.SlugDriveTypeSATASSD
 	}
 
+	fmt.Println("Got to UNKNOWN")
```

Comment:
purge debug print ? :)

Reply:
Debug prints are the seasoning of true champions ;)   Yeah will do.